controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -198,7 +198,8 @@ type DefaultMachinePoolSpec struct {
 	// +optional
 	InstanceType string `json:"instanceType,omitempty"`
 
-	// Autoscaling specifies auto scaling behaviour for this MachinePool.
+	// Autoscaling specifies auto scaling behaviour for the default MachinePool. Hosted clusters require that the number of
+	// worker nodes to autoscale min/max must be equal or multiple of the availability zones count.
```

Comment:
```suggestion
	// Autoscaling specifies auto scaling behaviour for the default MachinePool. min/max must be equal or multiple of the availability zones count.
```
user/customer don't know what Hosted clusters are so we shouldn't mention it.